arangosh/VPack/VPackFeature.cpp

```diff
@@ -230,17 +212,49 @@ void VPackFeature::start() {
     ofs.seekp(0);
   }
 
-  // write into stream
-  char const* start = buffer.data();
-  ofs.write(start, buffer.size());
+  bool const outputIsJson = (_outputType == "json" || _outputType == "json-hex" || _outputType == "json-pretty");
```

Comment:
`json-hex` isn't listed as output type on line 130:

```cpp
std::unordered_set<std::string> const outputTypes{{"json", "json-pretty", "vpack", "vpack-hex" }};
```

Reply:
True. I made a last minute change to it and removed it again. Now also removed the check for it.